### PR TITLE
README for curlz

### LINKED ISSUE #313: No documentation for curlz example
See https://openziti.discourse.group/t/troubles-with-using-the-golang-sdk/817 for details

There is no README or documentation for this example

### PR DESCRIPTION
**Pull request by gberl002**
Added functionality to accept an identity file
Added README for curlz example
closes #313

### CODE REVIEW

File: example/curlz/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+       cd <repo-root-dir>/sdk-golang/example/curlz/
+1. Run this script to create everything you need.
+
+       echo Copy the identity jwt into the current working directory
+       cp ../simple-server/simple-client.jwt .
+
+       echo Enroll the simple-client identity
+       ziti edge enroll --jwt simple-client.jwt
+
```
Comment (plorenz) [inline example/curlz/README.md:81]: The reference to `www.google.com` on this line and line 87 seem like they should be replaced with something about simple server

Comment (dovholuknf) [reply example/curlz/README.md:81]: it's probably better to replace all the 'www.google.com` with anything else, i'd agree... and then call out specifically that the service is offloading from the overlay at the router.

File: example/curlz/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+       echo Create the service
+       ziti edge create service www.google.com --configs "www.google.com.hostv1"
+       
+       echo Create an identity to make the dial request and enroll it
+       ziti edge create identity user curlz -a clients -o curlz.jwt
+       ziti edge enroll --jwt curlz.jwt
+       
+       echo Create service policies
+       ziti edge create service-policy www.google.com.dial Dial --service-roles '@www.google.com' --identity-roles '#clients'
+       ziti edge create service-policy www.google.com.bind Bind --service-roles '@www.google.com' --identity-roles '@MacBook-Pro.local-edge-router'
```
[the remark this answers is not in the document]
Comment (dovholuknf) [reply example/curlz/README.md:44]: or - maybe it makes sense to use an attribute here like `#curlz-example-offload` ?

[the remark this answers is not in the document]
Comment (dovholuknf) [reply example/curlz/README.md:44]: aaand ... or.... maybe say "if you used the quickstart, you'll have a tunneler enabled router. and then use the `#public` attribute?

[the remark this answers is not in the document]
Comment (gberl002) [reply example/curlz/README.md:44]: Yep, I knew I forgot something, I didn't intend to leave my hostname in there. I'll address these

File: example/curlz/README.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+       
+       echo Create edge router policies
+       ziti edge create edge-router-policy curlz-edge-router-policy --edge-router-roles '#all' --identity-roles '#clients,#servers'
+       ziti edge create service-edge-router-policy curlz-service-edge-router-policy --edge-router-roles '#all' --service-roles '@web.endpoint'
+       
+       echo Run policy advisor to check
+       ziti edge policy-advisor services
+1. Run the cURLz example for `web.endpoint`
+
+       ./curlz https://web.endpoint curlz.json
```
Comment (dovholuknf) [inline example/curlz/README.md:58]: you're only running `curlz` one time. what good was it to mention the 'non-ziti' service? Shouldn't you be making a curlz to https://www.google.com too?

Comment (dovholuknf) [reply example/curlz/README.md:58]: ooooh.. i get what you're going for now. hrmmm i just didn't get what it was all about at first here...

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**